packages/client/logic/shortcuts.ts

```diff
@@ -49,29 +49,31 @@ export function registerShortcuts() {
 
   const { escape, space, shift, left, right, up, down, enter, d, g, o } = magicKeys
   const shortcuts = new Map<string | Ref<Boolean>, ShortcutOptions>(
-    [
-      { key: and(space, not(shift)), fn: next, autoRepeat: true },
-      { key: and(space, shift), fn: prev, autoRepeat: true },
-      { key: and(right, not(shift), not(showOverview)), fn: next, autoRepeat: true },
-      { key: and(left, not(shift), not(showOverview)), fn: prev, autoRepeat: true },
-      { key: 'pageDown', fn: next, autoRepeat: true },
-      { key: 'pageUp', fn: prev, autoRepeat: true },
-      { key: and(up, not(showOverview)), fn: () => prevSlide(false), autoRepeat: true },
-      { key: and(down, not(showOverview)), fn: nextSlide, autoRepeat: true },
-      { key: and(left, shift), fn: () => prevSlide(false), autoRepeat: true },
-      { key: and(right, shift), fn: nextSlide, autoRepeat: true },
-      { key: and(d, not(drawingEnabled)), fn: toggleDark },
-      { key: and(o, not(drawingEnabled)), fn: toggleOverview },
-      { key: and(escape, not(drawingEnabled)), fn: () => showOverview.value = false },
-      { key: and(g, not(drawingEnabled)), fn: () => showGotoDialog.value = !showGotoDialog.value },
-      { key: and(left, showOverview), fn: prevOverviewPage },
-      { key: and(right, showOverview), fn: nextOverviewPage },
-      { key: and(up, showOverview), fn: upOverviewPage },
-      { key: and(down, showOverview), fn: downOverviewPage },
-      { key: and(enter, showOverview), fn: () => { go(currentOverviewPage.value); showOverview.value = false } },
-      ...customShortcuts,
-    ]
-      .map((options: ShortcutOptions) => [options.key, options]),
+    __SLIDEV_FEATURE_OVERRIDE_DEFAULT_SHORTCUTS__
```

Comment:
This may not need to be a compile-time flag (it was mainly for tree-shaking necessary packages).

You can read it from `import { configs } from '../env'`